### PR TITLE
Pacman Demo - Profile page and Monetization Pop-up

### PR DESCRIPTION
**Pull request by jcrabb-confluent**
The Profile page uses a new stream to show average stats per life per user

The Monetization pop up asks the user to watch an ad for a free life and sends their choices to a stream for farther analysis. 

You can watch the demo with Kai here;
https://videos.confluent.io/watch/aDr3o5V2ChD83AJaXLAgKZ

### CODE REVIEW

**Comment by jcrabb-confluent** [comment]
@gianlucanatali here is my PR, please review

File: streaming-pacman/config/demo.cfg.example
Comment (gianlucanatali) [inline streaming-pacman/config/demo.cfg.example:1]: Hi @jcrabb-confluent , we need this file to stay in the repo, otherwise people won't know how the config file looks like :) Can you add the file again? Thanks!

File: streaming-pacman/pacman/game/js/pacman.js
Comment (gianlucanatali) [inline streaming-pacman/pacman/game/js/pacman.js:46]: yes, but it is wrongly written also at usage time. If you want you can fix it in both places :)

File: streaming-pacman/pacman/game/js/pacman.js
Comment (gianlucanatali) [inline streaming-pacman/pacman/game/js/pacman.js:272]: What does this TODO means? Is it for a next iteration or you need to add something now? 
If is a future Idea maybe change from TODO to smg different

File: streaming-pacman/pacman/game/js/pacman.js
Comment (gianlucanatali) [inline streaming-pacman/pacman/game/js/pacman.js:282]: can we control this behavior from a variable? To enable/disable it?
The confirm function might be blocked by some adblock/phones etc...
For now would be enough to add a variable in the pacman.js file itself, at the very top and document it in the readme (so people can disable it if they need to)

File: streaming-pacman/pacman/game/js/scoreboard.js
Comment (gianlucanatali) [inline streaming-pacman/pacman/game/js/scoreboard.js:2]: Not used in this case but the call back should have it defined as that is what will receive.
I guess when we designed it we thought it could be useful. So I'd leave it there for now :)

Comment (gianlucanatali) [reply streaming-pacman/pacman/game/js/scoreboard.js:2]: here the reference: https://github.com/confluentinc/demo-scene/blob/ab824ce9f97952125518487a779753cb2549bac7/streaming-pacman/pacman/game/js/shared.js#L42

Comment (gianlucanatali) [reply streaming-pacman/pacman/game/js/scoreboard.js:2]: if you define the call back with one param only, it will just receive the value of the first one (highestScore)

File: streaming-pacman/terraform/aws/functions/src/main/java/com/gnatali/streaming/pacman/EventHandler.java
Comment (gianlucanatali) [inline streaming-pacman/terraform/aws/functions/src/main/java/com/gnatali/streaming/pacman/EventHandler.java:55]: Great! please remove the comment in the code, It will age quickly :)

**Review by gianlucanatali — CHANGES_REQUESTED** [review]

**Comment by gianlucanatali** [comment]
Another thing that I have noticed, is that the PROFILE Page is not showing anything if you enter it before dying. Maybe catch the error (i see one in the console) and show the scores as N/D or 0 ?

**Comment by gianlucanatali** [comment]
if you can give me write access to the repo/branch... so I can improve the popup with an ad if I have time ;)

**Comment by jcrabb-confluent** [comment]
You’re going to expose my lack of coding/github experience :D I tried to add you as a contributor to my fork, not sure if that is enough. 

I will look to work on these other pieces. FYI, TODO is a way I define low-priority notes to myself so I can ctrl-F and look for them later. Do you have suggestions for an alternative?

**Comment by jcrabb-confluent** [comment]
@rmoff I apologize - this has been out of sight, out of mind for me. I should be able to make the necessary changes with our "days off" tomorrow and Friday.

**Comment by rmoff** [comment]
I'm going to close this for now, please re-open as needed.

**Pull request closed without merging**
